### ISSUE TITLE
Fails on production build

### ISSUE THREAD

**Issue by thiagosolcia** [chadly/gatsby-remark-lottie#1]
Hi! thanks for this very handy plugin 😃 
The only problem is that it fails when I run a production build (see attachment). Any idea on how to fix it?

<img width="994" alt="gatsby-remark-lottie-issue" src="https://user-images.githubusercontent.com/33968018/97058709-7654c500-1586-11eb-8c59-2aa6c5d37a3f.png">

This is how I'm using it.

<img width="1436" alt="component" src="https://user-images.githubusercontent.com/33968018/97058974-288c8c80-1587-11eb-8b18-9b73b5bf5d93.png">

**Comment by chadly** [comment]
what does your `gatsby-config` look like?

**Comment by thiagosolcia** [comment]
like this
![gatsby-config](https://user-images.githubusercontent.com/33968018/97159121-4a963280-1772-11eb-9f37-f9d0d1f5c8f5.png)

**Comment by chadly** [comment]
try moving `gatsby-remark-lottie` first before `gatsby-remark-images`

**Comment by thiagosolcia** [comment]
Yes! that seems to do the trick :) Thank you! But now I'm getting another error. The plugin tries to load the json from the root public/ directory even though I have the json next to the .md file. I'm not sure what I did wrong here.

<img width="1274" alt="plugin-issue" src="https://user-images.githubusercontent.com/33968018/97373958-12e0d500-18af-11eb-8850-bd624483e977.png">

**Comment by mzane42** [comment]
@thiagosolcia did you find a solution for your last problem ? i'm stacked too

**Comment by chadly** [comment]
I'm not sure about the file path problem. I think that has to do with configuration outside of this plugin. One plugin I always make use of (and may help with this issue) is the [`gatsby-remark-copy-linked-files` plugin](https://www.gatsbyjs.com/plugins/gatsby-remark-copy-linked-files/). Maybe give that a try?
